### ISSUE TITLE
prisma studio start error

### ISSUE THREAD

**Issue by 1034935767** [prisma/nuxt-prisma#13]
I got this error. I use windows,.
```js
async function installStudio(directory) {
  try {
    const { spawn } = require("child_process");
    log(PREDEFINED_LOG_MESSAGES.installStudio.action);
    await spawn("npx", ["prisma", "studio", "--browser", "none"], {
      cwd: directory,
      shell: true // when use windows, this should be add 
    });
    logSuccess(PREDEFINED_LOG_MESSAGES.installStudio.success);
    return true;
  } catch (err) {
    logError(PREDEFINED_LOG_MESSAGES.installStudio.error);
    log(err);
    return false;
  }
}
```

```text
ERROR  [uncaughtException] spawn npx ENOENT                                                            09:51:02  

  at ChildProcess._handle.onexit (node:internal/child_process:286:19)
  at onErrorNT (node:internal/child_process:484:16)
  at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
```

**Comment by bbhxwl** [comment]
I can run it, but I cannot build it

**Comment by jharrell** [comment]
Duplicate of #12

**Comment by bbhxwl** [comment]
> I just want to mention that this added line does fix the issue, which I also encounter with a fresh install of Nuxt on Windows.

What specific line of code is it? My English is not good.

**Comment by JohanManders** [comment]
I temporary fixed it by adding ```shell: true``` to ```node_modules\@prisma\nuxt\dist\module.mjs```:

```js
async function installStudio(directory) {
  try {
    const { spawn } = require("child_process");
    log(PREDEFINED_LOG_MESSAGES.installStudio.action);
    await spawn("npx", ["prisma", "studio", "--browser", "none"], {
      cwd: directory,
      shell: true // when use windows, this should be add 
    });
    logSuccess(PREDEFINED_LOG_MESSAGES.installStudio.success);
    return true;
  } catch (err) {
    logError(PREDEFINED_LOG_MESSAGES.installStudio.error);
    log(err);
    return false;
  }
}
```

I hope an OS check gets added, to add this line is someone uses Windows.

Thanks to @1034935767

**Comment by bbhxwl** [comment]
> I temporary fixed it by adding `shell: true` to `node_modules\@prisma\nuxt\dist\module.mjs`:
> 
> ```js
> async function installStudio(directory) {
>   try {
>     const { spawn } = require("child_process");
>     log(PREDEFINED_LOG_MESSAGES.installStudio.action);
>     await spawn("npx", ["prisma", "studio", "--browser", "none"], {
>       cwd: directory,
>       shell: true // when use windows, this should be add 
>     });
>     logSuccess(PREDEFINED_LOG_MESSAGES.installStudio.success);
>     return true;
>   } catch (err) {
>     logError(PREDEFINED_LOG_MESSAGES.installStudio.error);
>     log(err);
>     return false;
>   }
> }
> ```
> 
> I hope an OS check gets added, to add this line is someone uses Windows.
> 
> Thanks to @1034935767

Where is this added to? I am different from you. I made an error using the drone CI CD, and I did not make any mistakes while developing on Windows.

**Comment by JohanManders** [comment]
@jharrell this specific problem is not fixed in version ```0.0.34```. I still have to use the workaround to let it work.
